firmware/application/receiver_model.hpp

```diff
@@ -26,49 +27,75 @@
 #include <cstddef>
 
 #include "app_settings.hpp"
+#include "max283x.hpp"
 #include "message.hpp"
 #include "rf_path.hpp"
-#include "max283x.hpp"
 #include "volume.hpp"
 
 // TODO: consider a base class for ReceiverModel & TransmitterModel.
 // There are multiple values that are actually shared by both.
 class ReceiverModel {
    public:
-    enum class Mode {
+    enum class Mode : uint8_t {
         AMAudio = 0,
         NarrowbandFMAudio = 1,
         WidebandFMAudio = 2,
         SpectrumAnalysis = 3,
         Capture = 4
     };
 
+    struct settings_t {
```

Comment:
Like for TX model, pulled all the fields into a struct that can be more easily initialized and copied. Allows us to pass settings overrides between apps instead of manipulating global state.